### PR TITLE
Settings & notifications

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

**Comment by andrewxhill** [comment]
Will need to do something with the Notifications page. I can stub in some basic ones that are more or less static (mnemonic created, node created, node started) and then we can build around it as a later pr

File: App/Navigation/AppNavigation.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import TextileManager from '../Containers/TextileManager'
 import OnboardingCheck from '../Containers/OnboardingCheck'
 import AndrewPairingInvite from '../SB/views/AndrewDeviceInvite'
 import AndrewThreadInvite from '../SB/views/AndrewThreadInvite'
+import AccountNavigation from './AccountNavigation'
```
Comment (asutula) [inline App/Navigation/AppNavigation.js]: Don't need this here.

File: App/Navigation/PhotosNavigation.js [hunk 2]
```diff
@@ -2,21 +2,23 @@ import React from 'react'
 import { StackNavigator, TabNavigator, TabBarBottom } from 'react-navigation'
 import { Image } from 'react-native'
 import TextilePhotos from '../Containers/TextilePhotos'
-import InfoView from '../Containers/InfoView'
 
 import styles, {headerTintColor} from './Styles/NavigationStyles'
+import AccountNavigation from './AccountNavigation'
```
Comment (asutula) [inline App/Navigation/PhotosNavigation.js]: Unused I think.

File: App/Redux/AuthRedux.ts [hunk 3]
```diff
@@ -20,6 +20,9 @@ const actions = {
   logInRequest: createAction('LOG_IN_REQUEST', resolve => {
     return (username: string, password: string) => resolve({ username, password })
   }),
+  logOutRequest: createAction('LOG_OUT_REQUEST', resolve => {
```
Comment (asutula) [inline App/Redux/AuthRedux.ts]: Don't see any 'log out succes' action, but do we need it so we can clear out the `CafeTokens` stored in `AuthRexux`?

File: App/Navigation/PhotosNavigation.js (hunk 2 above)
Comment (andrewxhill) [reply App/Navigation/PhotosNavigation.js]: It is, it's part of the root nav at the bottom

File: App/Redux/AuthRedux.ts (hunk 3 above)
Comment (andrewxhill) [reply App/Redux/AuthRedux.ts]: I didn't complete it, figured it needs its own PR later. I commented out the Lock Screen button in the settings for now. The reason was that we need a few things to make it work,

- [ ] Goodbye screen (optional)
- [ ] When logged out after loggin in, we might want to skip those three intro screens before the login
- [ ] We currently force the user to retake their profile pic after any login, we prob want to skip that or at least ensure it works

**Comment by andrewxhill** [comment]
hmm.. just merged master into it and it doesn't like something. looking into it

**Comment by andrewxhill** [comment]
okay. let's do it

**Pull request merged**